examples/gallery/embellishments/solar.py

```diff
@@ -15,39 +15,24 @@
 import pygmt
 
 fig = pygmt.Figure()
-# Create a figure showing the global region on a Mollweide projection
-# Land color is set to dark green and water color is set to light blue
-fig.coast(region="d", projection="W0/15c", land="darkgreen", water="lightblue")
-# Set a time for the day-night terminator and twilights, 17:00 UTC on
-# January 1, 2000
+# Create a global map with central longitude 0° East ("d") using a Mollweide ("W")
+# projection with a width of 15 centimeters
+fig.basemap(region="d", projection="W15c", frame=True)
+fig.coast(land="darkgreen", water="lightblue", shorelines="1/1p,gray50")
+
+# Set a time for the day-night terminator and twilights to 17:00 UTC on January 1, 2000
 terminator_datetime = datetime.datetime(
```

Comment:
Could you please also try to find a short name for this variable? `terminator_datetime=terminator_datetime` is really so long.